src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_pubsub.v1.json.baseline

```diff
@@ -67,7 +67,7 @@ authorize(function(authClient) {
   var request = {
     // REQUIRED: The resource for which the policy is being specified. `resource` is usually specified as
     // a path. For example, a Project resource is specified as `projects/{project}`.
-    resource: '',  // TODO: Update placeholder value.
+    resource_: '',  // TODO: Update placeholder value.
```

Comment:
This is what the client library expects, cloudresourcemanager has a field `resource`, and the Node.js library maps requestBody to the `resource` parameter. To resolve this conflict, the library owners have decided to rename any parameters called `resource` to `resource_`.

So, this is a deliberate design choice in the client libraries, so this is correct.